docs/data-sources/warehouse.md

```diff
@@ -29,13 +29,13 @@ output "warehouse_region" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
 
+- `id` (String) Warehouse ID
```

Comment:
Just noticed I left a typo on L35. `Role Name` instead of `Warehouse Name`.

Are both optional? Assuming you have to provide at least one of them.

Reply:
In this case, they are mutually exclusive. There might be a better way to do this, but at the very least, it should include a validator to ensure at least one is set.